src/plugins/auto_batch/tests/functional/behavior/ov_plugin/auto_batching_tests.cpp

```diff
@@ -37,23 +37,23 @@ INSTANTIATE_TEST_SUITE_P(smoke_AutoBatching_test,
                          ::testing::Combine(::testing::Values(std::string(ov::test::utils::DEVICE_BATCH) + ":" +
                                                               ov::test::utils::DEVICE_TEMPLATE),
                                             ::testing::Values(DefaultParameter{CONFIG_KEY(AUTO_BATCH_TIMEOUT),
-                                                                               InferenceEngine::Parameter{"1000"}})),
+                                                                               ov::Any{"1000"}})),
                          DefaultConfigurationTest::getTestCaseName);
 
 INSTANTIATE_TEST_SUITE_P(smoke_AutoBatching_test_string,
                          DefaultConfigurationTest,
                          ::testing::Combine(::testing::Values(std::string(ov::test::utils::DEVICE_BATCH) + ":" +
                                                               ov::test::utils::DEVICE_TEMPLATE),
                                             ::testing::Values(DefaultParameter{ov::auto_batch_timeout.name(),
-                                                                               InferenceEngine::Parameter{"1000"}})),
+                                                                               ov::Any{"1000"}})),
```

Comment:
let's use `ov::auto_batch_timeout(1000)` - it will allow remove `DefaultParameter`